packages/format-library/src/link/modal-screens/link-settings-screen.native.js

```diff
@@ -213,6 +217,15 @@ const LinkSettingsScreen = ( {
 							onValueChange={ setOpensInNewWindows }
 							separatorType={ 'fullWidth' }
 						/>
+						<BottomSheet.SwitchCell
+							icon={ external }
+							label={ __(
+								'Search engines should ignore this link (mark as nofollow)'
```

Comment:
This is a bit long for mobile so maybe we could use just the part "No follow" as suggested in [this comment](https://github.com/WordPress/gutenberg/pull/48722/files#r1154294933).

<kbd><img src="https://user-images.githubusercontent.com/4885740/229571889-d85b5bd6-0701-48e5-91eb-7f7d790c1c14.png" width=200 /></kbd>